README.md

```
@@ -2,51 +2,124 @@
 
-### Database
+## API Reference
 
-```bash
-docker-compose -f docker/docker-compose.yml up -d
-```
+The backend provides a RESTful API. Key endpoints include:
 
-## API
+| Method | Endpoint | Description |
+|--------|----------|-------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)